software/kernel/boards/ambient_module/Makefile

```diff
@@ -1,12 +1,10 @@
-# Makefile for building the tock kernel for the Hail platform
```

Comment:
They're all already updated anyways. Delete them when making new copies if you so desire.